automated/linux/kvm-unit-tests/kvm-unit-tests.sh

```diff
@@ -48,9 +50,14 @@ kvm_unit_tests_run_test() {
 
 kvm_unit_tests_build_test() {
     info_msg "git clone kvm unit tests ..."
-    git clone https://git.kernel.org/pub/scm/virt/kvm/kvm-unit-tests.git --depth 1
+    git clone https://git.kernel.org/pub/scm/virt/kvm/kvm-unit-tests.git
     # shellcheck disable=SC2164
     cd kvm-unit-tests
+    if [ "${BRANCH}" = "5.2" ]; then
```

Comment:
I don't understand why this is hard coded. Can we just have a GIT_REF parameter and "git checkout" it regardless of what it is? What am I missing?

Reply:
@danrue Addressed your comments with GIT_REF.